src/raft.rs

```diff
@@ -836,6 +872,83 @@ impl<T: Storage> RaftCore<T> {
         true
     }
 
+    // Pack MsgAppend according to forward info, and send it to target peer.
+    fn send_forward(
```

Comment:
Questions: is it used for leader to resend information that follower fails to replicated? If that maybe resend_forward may be better?